src/Compilers/VisualBasic/Portable/Scanner/Scanner.vb

```diff
@@ -1432,8 +1481,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
             Dim Here = Index
```

Comment:
While you're in here. Could you rename the local 'Here' to "offset". I've never found the name 'Here' to be very informative/intuitive.